conjure-java-core/src/main/java/com/palantir/conjure/java/Options.java

```diff
@@ -163,6 +163,15 @@ default boolean jetbrainsContractAnnotations() {
         return false;
     }
 
+    /**
+     * If set to true, static factory methods ('of') will be excluded from generated objects with one or more fields.
+     * Note that for objects without any fields, this will still generate the static factory method.
+     */
+    @Value.Default
+    default boolean excludeStaticFactoryMethods() {
```

Comment:
Preferring this naming instead of `omitGeneratedOfMethods` because I think it's more descriptive.